Problems/prob134/specification.md

```diff
@@ -0,0 +1,11 @@
+---
+Title:    Train Traffic Rescheduling
+Proposer: Quentin Cappart and Pierre Schaus 
```

Comment:
Can you change this to the following, please?
```
Proposer:
- Quentin Cappart
- Pierre Schaus
```

Otherwise, CSPLib will treat it as one long name instead of two separate names.